doc/user_doc/vic_installation/vch_installer_options.md

```diff
@@ -47,178 +53,263 @@ If you are deploying a virtual container host on vCenter Server, specify a usern
+**NOTE** If your vSphere environment uses untrusted, self-signed certificates, you can run `vic-machine create` without the `--thumbprint` option by using the `--force` option. However, running `vic-machine create` with the `--force` option rather than providing the certificate thumbprint is not recommended, because it permits man-in-the-middle attacks to go undetected.
 
-You pass the name of the distributed port group to the `bridge-network` option. Each virtual container host requires its own distributed port group. 
+To obtain the thumbprint of the vCenter Server or ESXi host certificate, run `vic-machine create` without the specifying the `--thumbprint` or `--force` options. The deployment of the virtual container host fails, but the resulting error message includes the required certificate thumbprint. You can copy the thumbprint from the error message and run vic-machine create again, including the `thumbprint` option.
 
-**IMPORTANT** 
-- Do not assign the same `bridge-network` distributed port group to multiple virtual container hosts. Sharing a distributed port group between virtual container hosts might result in multiple container VMs being assigned the same IP address. 
-- Do not use the `bridge-network` distributed port group as the target for any of the other `vic-machine create` networking options.
+<pre>--thumbprint <i>certificate_thumbprint</i></pre>
 
-If you specify an invalid network name, `vic-machine create` fails and suggests valid networks.
+<a name="security"></a>
+## Security Options ##
 
-The `bridge-network` option is **optional** when you are deploying a virtual container host to an ESXi host with no vCenter Server. In this case, if you do not specify `bridge-network`, `vic-machine` creates a  virtual switch and a port group that each have the same name as the virtual container host. You can optionally specify this option to assign an existing port group for use as the bridge network for container VMs. You can also optionally specify this option to create a new virtual switch and port group that have a different name to the virtual container host.
+When you deploy a virtual container host, you specify the type of authentication to use when Docker clients connect to that virtual container host. 
 
-<pre>--bridge-network <i>distributed_port_group_name</i></pre>
+- Two-way authentication with trusted auto-generated TLS certificates that are signed by a Certificate Authority (CA). Specify the [`tls-cname`](#tls-cname) option when you deploy the virtual container host.
+- Server-side authentication with auto-generated, untrusted TLS certificates that are not signed by a CA, with no client-side verification. Specify the [`no-tlsverify`](#no-tlsverify) option when you deploy the virtual container host.
```

Comment:
Pulled all of the TLS overview-type content out of the topic for 0.7, pending rewrite.